scripts/publish_pr.py

```
@@ -0,0 +1,389 @@
+        pr_body = create_placeholder_pr_body(issue_id, issue_title, issue_body)
+
+    # Append Closes directive if not already present
+    if f"Closes #{issue_id}" not in pr_body:
+        pr_body += f"\n\nCloses #{issue_id}"
+
+    # Create PR
+    print("Creating PR...")
+    pr_url = create_pr(owner, repo, branch, default_branch, f"[Issue #{issue_id}] {issue_title}", pr_body)
+
+    if pr_url:
+        print(f"\n✓ PR created successfully!")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)